### PR TITLE
Install and configure phpmyadmin

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 2269

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-87ac27ac35/how-to/). :bow:

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+product_url: cloud-servers
+---
+
+phpMyAdmin&reg; is a free and open source administration tool for MySQL&reg;
+and MariaDB&reg;. As a portable web application written primarily in PHP,
+phpMyAdmin has become one of the most popular MySQL administration tools,
+especially for web hosting services.
+
+This article describes how to install and configure phpMyAdmin on your
+web server for CentOS&reg; 7, Red Hat&reg; Enterprise Linux&reg; 7, and
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest adding (RHEL) acronym.

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+
+**CentOS and RHEL 6**
+
+    service httpd status
+
+**CentOS and RHEL 7**
+
+    systemctl status httpd
+
+You should now be able to view phpMyAdmin through a web browser.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest adding ", as shown in the following image:"

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 3]
```diff
@@ -0,0 +1,648 @@
...
+
+**CentOS and RHEL 6**
+
+    service nginx status
+
+**CentOS and RHEL 7**
+
+    systemctl status nginx
+
+You should now be able to view phpMyAdmin through a web browser.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Same as previous comment.

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 4]
```diff
@@ -0,0 +1,648 @@
...
+#### Apache web server
+
+The installation process adds the phpMyAdmin Apache configuration file to the
+**/etc/apache2/conf-enabled/** directory, where it is read automatically. The
+only thing you need to do is to enable the `mbstring` PHP extension, which
+you can do by running the following command:
+
+    sudo phpenmod mbstring
+
+After installing phpMyAdmin, the package configuration screen displays.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Same comment.

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 5]
```diff
@@ -0,0 +1,648 @@
...
+Use the space bar to select **apache2**, press **Tab** to select
+**Ok**, and then press **Enter**.
+
+<img src="{% asset_path cloud-servers/install-and-configure-phpmyadmin/phpmyadmin-package-configuration-select-apache2.png %}" />
+
+The installation process continues until another configuration screen displays
+that confirms if you want to configure your database for phpMyAdmin by using
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest "...that prompts you to confirm that..."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 6]
```diff
@@ -0,0 +1,648 @@
...
+To use phpMyAdmin to administer a MySQL database hosted on another server,
+open **/etc/phpmyadmin/config.inc.php** in a text editor and then edit
+the following line:
+
+| Old line | New line |
+| --- | --- |
+| `$cfg['Servers'][$i]['host'] = '$dbserver';` | `$cfg['Servers'][$i]['host'] = '192.168.71.21';` |
+
+**Note:** Replace `$dbserver` with the actual remote database server name or
+IP address. Also, be sure that the phpMyAdmin host has permissions to access
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest "...ensure..."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 7]
```diff
@@ -0,0 +1,648 @@
...
+the remote database.
+
+The other configuration file that you must edit is
+**/etc/phpmyadmin/apache.conf**. This file is linked symbolically to
+**/etc/apache2/conf-available/phpmyadmin.conf**, and after enabled, is used
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest ". After it is enabled, it is used..."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 8]
```diff
@@ -0,0 +1,648 @@
...
+Then reload the Apache web server by running the following command:
+
+    systemctl reload apache2
+
+Check the status of the service to ensure that it is functioning as
+expected by running the following command:
+
+    system status apache2
+
+You should now be able to view phpMyAdmin through a web browser.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest adding "..., as shown in the following image:"

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 9]
```diff
@@ -0,0 +1,648 @@
...
+
+<img src="{% asset_path cloud-servers/install-and-configure-phpmyadmin/phpmyadmin-browser.png %}" />
+
+#### NGINX web server
+
+After installing phpMyAdmin, the package configuration screen displays.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Same as above.

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 10]
```diff
@@ -0,0 +1,648 @@
...
+Use the space bar to select **apache2**, press **Tab** to select
+**Ok**, and then press **Enter**.
+
+<img src="{% asset_path cloud-servers/install-and-configure-phpmyadmin/phpmyadmin-package-configuration-select-apache2.png %}" />
+
+The installation process continues until another configuration screen displays
+that confirms if you want to configure your database for phpMyAdmin by using
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest "...that asks you to confirm if..."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 11]
```diff
@@ -0,0 +1,648 @@
...
+             fastcgi_pass unix:/run/php/php7.0-fpm.sock;
+             fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+             include fastcgi_params; include snippets/fastcgi-php.conf;
+       }
+      }
+    }
+
+Your phpMyAdmin files are located in the **/usr/share/phpmyadmin/** directory.
+The configuration above tells NGINX that if visitors enter
+**http://ip_address/phpmyadmin** in the browser address bar to find the
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest "...bar, it should find..."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 12]
```diff
@@ -0,0 +1,648 @@
...
+
+**RHEL and CentOS 6**
+
+    service nginx status
+
+**RHEL and CentOS 7**
+
+    systemctl status nginx
+
+You should now be able to view phpMyAdmin through a web browser.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest adding "..., as shown in the following image:"

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 13]
```diff
@@ -0,0 +1,648 @@
...
+
+<img src="{% asset_path cloud-servers/install-and-configure-phpmyadmin/phpmyadmin-browser.png %}" />
+
+### Configure additional security (optional)
+
+`htpasswd` is used to create and update the flat files used to store
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest "...the flat files that store..."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 14]
```diff
@@ -0,0 +1,648 @@
...
+usernames and passwords for the basic authentication of HTTP users. If
+`htpasswd` cannot access a file, such as not being able to write to the
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest "If `htpasswd` cannot access a file (cannot write to the output file or read the file in order to update it), it returns an error status and makes no changes."

File: content/cloud-servers/install-and-configure-phpmyadmin.md [hunk 15]
```diff
@@ -0,0 +1,648 @@
...
+output file or not being able to read the file in order to update it,
+it returns an error status and makes no changes.
+
+Use the steps in the following sections to set up basic authentication on
+a web server running phpMyAdmin.
+
+#### Apache web server
+
+By default, Apache does not allow the use of `.htaccess`. You must configure
+Apache to allow `.htaccess` based authentication.
```
Comment (RackerKate) [inline content/cloud-servers/install-and-configure-phpmyadmin.md]: Suggest `.htaccess`-based.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-89f6c66750/how-to/)

**Pull request merged**